samples/algorithms/chsh-game/Game.qs

```diff
@@ -143,4 +141,22 @@ namespace Microsoft.Quantum.Samples.CHSHGame {
             return MeasureBobQubit(bobBit, bobQubit) == MeasureAliceQubit(aliceBit, aliceQubit);
         }
     }
+
+    /// # Summary
+    /// Plays the optimal classical strategy for the CHSH game: both Alice and Bob both always
+    /// output 0. This should result in success 75% of the time.
```

Comment:
FYI: "**both** Alice and Bob **both** always".
(one **both** seems to me redundant)